node/silkworm/downloader/rpc/hand_shake.hpp

```diff
@@ -0,0 +1,33 @@
+/*
+   Copyright 2021 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef SILKWORM_HAND_SHAKE_HPP
+#define SILKWORM_HAND_SHAKE_HPP
+
+#include <silkworm/chain/identity.hpp>
+#include <silkworm/downloader/internals/types.hpp>
+#include <silkworm/downloader/sentry_client.hpp>
+
+namespace silkworm::rpc {
+
+  class HandShake : public rpc::UnaryCall<sentry::Sentry, google::protobuf::Empty, sentry::HandShakeReply> {
```

Comment:
Do we really need to create a whole source file for such small classes ?
Can't we group them together ?